Cartfile

```diff
@@ -0,0 +1,2 @@
+github "FLIR/SnapKit" "flir/add-compatibility-version"
```

Comment:
Currently SnapKit 5.0 doesn't work with Carthage, this branch does and is being integrated in https://github.com/SnapKit/SnapKit/pull/588

Reply:
Thanks Ian @Author . I am sorry it took me too long. I did not expect any attention. 

Though now I need to look whether this is all what needs to be done to support Carthage.
If you worked with Carthage you can direct to instructions, I will try to do it as fast as possible.

Reply:
Thanks so much for merging it! Were you able to double check that this was enough?

Reply:
Yeah I checked it against Carthage support pages and tried to build.
All went smooth so I merged it. Thanks again for the help :)